torch/_inductor/autotune_process.py

```diff
@@ -19,9 +21,13 @@
 from .utils import do_bench
 from .virtualized import V
 
-DEBUG = False
+
 EXIT_HANDLER_REGISTERED = False
 
+log = logging.getLogger(__name__)
+if os.environ.get("TORCHINDUCTOR_MAX_AUTOTUNE_DEBUG", "0") == "1":
+    log.setLevel(logging.DEBUG)
```

Comment:
We don't need this. By default, log level is set to WARNING. If you want to enable debug log for a module like this, override TORCH_LOGS environment variable should work:
```
TORCH_LOGS=+torch._inductor.autotune_process python xxx
```

Reply:
Ah. Thanks!